tools/docker-network/tests/committeerotation_test.go

```diff
@@ -0,0 +1,118 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/iotaledger/hive.go/ierrors"
+	iotago "github.com/iotaledger/iota.go/v4"
+	"github.com/stretchr/testify/require"
+)
+
+var ValidatorContainerNames = map[string]string{
+	"V1": "docker-network-inx-validator-1-1",
+	"V2": "docker-network-inx-validator-2-1",
+	"V3": "docker-network-inx-validator-3-1",
+	"V4": "docker-network-inx-validator-4-1",
+}
+
+var ClientURLs = map[string]string{
+	"V1":    "http://localhost:8050",
+	"V2":    "http://localhost:8060",
+	"V3":    "http://localhost:8070",
+	"V4":    "http://localhost:8040",
+	"node5": "http://localhost:8090",
+}
+
+func Test_SmallerCommittee(t *testing.T) {
+	d := NewDockerTestFramework(t, ValidatorContainerNames, ClientURLs,
+		WithProtocolParametersOptions(
+			iotago.WithTimeProviderOptions(5, time.Now().Unix(), 10, 4),
+			iotago.WithLivenessOptions(30, 30, 2, 4, 8),
+			iotago.WithTargetCommitteeSize(4),
+		))
+	defer d.Stop()
+
+	err := d.Run()
+	require.NoError(t, err)
+
+	err = d.WaitUntilSync()
+	require.NoError(t, err)
+
+	node := d.GetRandomNode()
+	status, err := getNodeStatus(node)
+	require.NoError(t, err)
+
+	currentEpoch := node.CommittedAPI().TimeProvider().EpochFromSlot(status.LatestAcceptedBlockSlot)
+
+	// stop validator 2
+	err = d.StopContainer("V2")
+	require.NoError(t, err)
+
+	d.AssertCommitteeSelection(currentEpoch+2, 3)
```

Comment:
As I commented on the function: maybe it would be better to assert the committee explicitly to make sure the members are in there.